queries.sql

```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
-
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN '1-1-2016' AND '1-1-2019';
```

Comment:
- [ ] You made a good move here :+1: . But let's assume an animal was born on the `5-3-2019` or other months after but still in 2019 year,  this query won't fetch them. 
And remember that this query should fetch all animals between 2016 and 2019.
I suggest you end your check on the last day and last month of 2019. By doing so, your query will fetch all animals born between 2016 - 2019.

Reply:
Thank you @mmsesay .I have worked on the requested changes